src/VerticalInterpPdf.cc

```diff
@@ -48,6 +33,11 @@ VerticalInterpPdf::VerticalInterpPdf(const char *name, const char *title, const
     assert(0);
   }
 
+#if ROOT_VERSION_CODE >= ROOT_VERSION(6,32,0)
+  // With ROOT 6.32, RooFit can do the type checking for us
+  _funcList.addTyped<RooAbsReal>(inFuncList);
+  _funcList.addTyped<RooAbsReal>(inCoefList);
```

Comment:
this is a typo, should be: 
`_coefList.addTyped<RooAbsReal>(inCoefList);` instead